src/main/webapp/css/pull-request-monitoring.css

```diff
@@ -233,7 +233,7 @@
   font-family: 'Material Icons';
   font-style: normal;
   font-weight: 400;
-  src: url(https://fonts.gstatic.com/s/materialicons/v29/2fcrYFNaTjcS6g4U3t-Y5ZjZjT5FdEJ140U2DJYC3mY.woff2) format('woff2');
+  src: url(../fonts/2fcrYFNaTjcS6g4U3t-Y5ZjZjT5FdEJ140U2DJYC3mY.woff2) format('woff2');
```

Comment:
Why are you using a font here? Don't we have the icon ready as a sprite?

Reply:
Thats the material icons font. What do you mean with 'sprite'?

Reply:
I think @fqueiruga included the material icons as sprite in Jenkins core and you can select the corresponding icon by referencing the sprite and the ID within that sprite. 

Here are all supported icons:
https://github.com/jenkinsci/jenkins/tree/master/war/src/main/webapp/images/material-icons